.Gruntfile.babel.js

```diff
@@ -417,6 +473,38 @@ function flow (options = {}) {
   }
 }
 
+function svgLoader (options) {
+  // Based on "rollup-plugin-vue-inline-svg" but better:
+  // - fix sourcemaps bug on original plugin
+  // - remove useless attrs and add global CSS class.
```

Comment:
Very nice. You get several kudos here:

1. Excellent comments, they explain clearly where you got the inspiration for this from, and they explain your modifications/improvements
2. You avoided adding yet another unnecessary dependency to our codebase
3. Down below you and up above you've linked to relevant Github issues as appropriate

Bravo! 👏

Reply:
yey thanks @taoeffect!